modules/mono/glue/GodotSharp/GodotSharp/Core/Variant.cs

```diff
@@ -120,6 +120,14 @@ public void Dispose()
         }
     }
 
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public static Variant From<[MustBeVariant] T>(in T from) =>
+        CreateTakingOwnershipOfDisposableValue(VariantUtils.CreateFrom(from));
+
+    [MethodImpl(MethodImplOptions.AggressiveInlining)]
+    public T As<[MustBeVariant] T>() =>
+        VariantUtils.ConvertTo<T>(NativeVar.DangerousSelfRef);
```

Comment:
I think this is enough to close https://github.com/godotengine/godot-proposals/issues/5762